src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -134,6 +141,53 @@ class HaConfigDashboard extends LitElement {
                     .pages=${configSections.dashboard}
                   ></ha-config-navigation>
                 </ha-card>`}
+          <div class="tips">
+            <ha-svg-icon .path=${mdiLightbulbOutline}></ha-svg-icon>
+            <span class="tip-word">Tip!</span>
+            <span class="text"
+              >${this.hass.localize(
+                "ui.panel.config.tips.join",
+                "forums",
+                html`<a
+                  href="https://community.home-assistant.io/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Forums</a
+                >`,
+                "twitter",
+                html`<a
+                  href="https://www.home-assistant.io/twitter/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Twitter</a
+                >`,
+                "discord",
+                html`<a
+                  href="https://www.home-assistant.io/join-chat/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Chat</a
+                >`,
+                "blog",
+                html`<a
+                  href="https://www.home-assistant.io/blog/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Blog</a
+                >`,
+                "newsletter",
+                html`
+                  <a
+                    href="https://www.home-assistant.io/newsletter/"
+                    target="_blank"
+                    rel="noreferrer"
+                    >Newsletter</a
+                  ></span
```

Comment:
These kinds of splits, while correct does not look that great :smile:

Reply:
ah its orphaned